handbook/engineering/campaigns/index.md

```diff
@@ -41,7 +41,21 @@ Users provide the code to make the change, we provide the plumbing to turn it in
 
 ## Goals
 
-[WIP Goals (internal only)](https://docs.google.com/document/d/1ADmpfJuVhprHl8Eimuj4fMiVr1kCRVLZpt98HLr9618/edit)
+The short-term goal and primary focus of the team is to *deliver campaigns to all users*. Our milestones toward that goal:
+
+1. Someone on the Campaigns team makes one meaningful change to at least two repositories.
+1. A Sourcegraph engineer (not on this team) makes one meaningful change to at least two repositories.
+1. Reach out to one customer, summarizing the changes in the new Campaigns workflow, to give them a heads-up.
+   - Hypothesis: they will reply within 3 days
+   - Hypothesis: they will engage within 5 days
+1. Reach out to two other customers, as in previous step.
```

Comment:
I assume we wait until we get one customer to reply and engage before moving on to this step (would be nice for this to be explicit)

Reply:
Will do.